src/app/views/sidebar/sample-queries/SampleQueries.tsx

```diff
@@ -123,7 +133,7 @@ export class SampleQueries extends Component<ISampleQueriesProps, any> {
                 tooltipProps={{
                   onRenderContent: () => (
                     <div style={{ paddingBottom: 3 }}>
-                      <FormattedMessage id={signInText} />
+                      <FormattedMessage id={"signInText"} />
```

Comment:
why were the quotes added. i think its referring to the variable `signInText` above.
![image](https://user-images.githubusercontent.com/55033656/123853906-d8fb1f00-d8d2-11eb-985f-42c8c0d28d23.png)

Reply:
Nice catch!!